src/config.rs

```diff
@@ -69,6 +189,69 @@ impl Options for DefaultOptions {
     }
 }
 
+/// A SizeLimit that restricts serialized or deserialized messages from
+/// exceeding a certain byte length.
+#[derive(Copy, Clone)]
+pub struct Bounded(pub u64);
```

Comment:
Should probably be named "UpperBound" or "MaxSized"

Reply:
I prefer `Bounded` and its been that way for a while so I'll leave it.